code/src/java/plugin/primitive/deity/PantheonToken.java

```diff
@@ -1,110 +0,0 @@
-/*
- * Copyright 2009 (C) Thomas Parker <thpr@users.sourceforge.net>
- *
- * This library is free software; you can redistribute it and/or
- * modify it under the terms of the GNU Lesser General Public
- * License as published by the Free Software Foundation; either
- * version 2.1 of the License, or (at your option) any later version.
- *
- * This library is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
- * Lesser General Public License for more details.
- *
- * You should have received a copy of the GNU Lesser General Public
- * License along with this library; if not, write to the Free Software
- * Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
- */
-package plugin.primitive.deity;
-
-import java.util.Collection;
-
-import pcgen.cdom.base.CDOMReference;
-import pcgen.cdom.base.Converter;
-import pcgen.cdom.base.PrimitiveFilter;
-import pcgen.cdom.enumeration.GroupingState;
-import pcgen.cdom.enumeration.ListKey;
-import pcgen.cdom.enumeration.Pantheon;
-import pcgen.core.Deity;
-import pcgen.core.PlayerCharacter;
-import pcgen.rules.context.LoadContext;
-import pcgen.rules.persistence.token.PrimitiveToken;
-
-public class PantheonToken implements PrimitiveToken<Deity>, PrimitiveFilter<Deity>

```

Comment:
Is there a conversion path for the use of the PANTHEON and SPELLTYPE primitives?